TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/support/SocketConnectionObservable.java

```diff
@@ -68,4 +69,16 @@ public void run() {
         }
     };
 
+    private final Emitter.Listener disconnectionListener = new Emitter.Listener() {
```

Comment:
When a `Bot` or `Human` is disconnected it disconnects whoever it is communicating with too. This is to ensure that the UI will be updated in the event that a `disconnection` event occurs.